js/main.js

```diff
@@ -7,4 +7,177 @@ const closeMobileMenu = () => {
+  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+  popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  image: './images/Snap2.png',
+  imageMobile: './images/Snapshoot2.png',
+  imagePopup: './images/Snap2.png',
+  imageMobilePopup: './images/Snapshoot2.png',
+  seeLive: 'https://tariktesfa.github.io/My-portfolio/',
+  seeSource: 'https://github.com/tariktesfa/My-portfolio',
+},
+{
+  title: 'Facebook 360',
+  exprience: ['FACEBOOK', 'Back End Dev', '2015'],
+  desription: 'Exploring the future of media in Facebook first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+  popDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+  popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  image: './images/Snap3.png',
+  imageMobile: './images/Snapshoot3.png',
+  imagePopup: './images/Snap3.png',
+  imageMobilePopup: './images/Snapshoot3.png',
+  seeLive: 'https://tariktesfa.github.io/My-portfolio/',
+  seeSource: 'https://github.com/tariktesfa/My-portfolio',
+},
+{
+  title: 'Uber Navigation',
+  exprience: ['UBER', 'Lead Developer', '2018'],
+  desription: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
```

Comment:
- [ ] Please make sure the popup follows the design for both desktop and mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins)😉.
- Your popup for mobile
![screenshot (14)](https://user-images.githubusercontent.com/44624138/146240986-4f331d3f-81c7-4ec2-b05c-151bdbef000f.png)
- Figma popup for mobile
![screenshot (15)](https://user-images.githubusercontent.com/44624138/146241204-2afd5e36-95f3-46d7-8d5a-074ccdff583a.png)

Reply:
I have fixed the issue with the mobile view.